backend/Dockerfile

```diff
@@ -1,12 +1,16 @@
-FROM alpine:3.11
+FROM alpine:3.12
 
-RUN \ 
-apk add --no-cache npm python ffmpeg && \
+ENV UID=991 GID=991
+
+RUN \
+apk add --no-cache npm python2 ffmpeg tini su-exec && \
```

Comment:
We do not need to install tini here. You can pass ``--init`` to docker run or similar to achieve the same result.